app/scripts/modules/ecs/serverGroup/details/resize/resizeCapacity.component.html

```diff
@@ -0,0 +1,88 @@
+<div ng-if="$ctrl.command.advancedMode">
+  <div class="form-group">
+    <div class="col-md-12">
+      <p>Sets up auto-scaling for this server group.</p>
+      <p>To disable auto-scaling, use the <a href ng-click="$ctrl.command.advancedMode = false">Simple Mode</a>.</p>
+    </div>
+  </div>
+  <div class="form-group">
+    <div class="col-md-2 col-md-offset-3"><b>Min</b></div>
+    <div class="col-md-2"><b>Max</b></div>
+    <div class="col-md-2"><b>Desired</b></div>
+  </div>
+  <div class="form-group">
+    <div class="col-md-3 sm-label-right">Current</div>
+    <div class="col-md-2"><input type="number"
+                                 class="form-control input-sm"
+                                 ng-model="$ctrl.currentSize.min"
+                                 readonly/>
+    </div>
+    <div class="col-md-2"><input type="number"
+                                 class="form-control input-sm"
+                                 ng-model="$ctrl.currentSize.max"
+                                 readonly/>
+    </div>
+    <div class="col-md-2"><input type="number"
+                                 class="form-control input-sm"
+                                 ng-model="$ctrl.currentSize.desired"
+                                 readonly/>
+    </div>
+  </div>
+  <div class="form-group">
+    <div class="col-md-3 sm-label-right">Resize to</div>
+    <div class="col-md-2"><input type="number"
+                                 class="form-control input-sm"
+                                 ng-model="$ctrl.command.min"
+                                 min="0"
+                                 required
+                                 max="{{$ctrl.command.max}}"/></div>
+    <div class="col-md-2"><input type="number"
+                                 class="form-control input-sm"
+                                 ng-model="$ctrl.command.max"
+                                 required
+                                 min="{{$ctrl.command.min}}"/></div>
+    <div class="col-md-2"><input type="number"
+                                 class="form-control input-sm"
+                                 ng-model="$ctrl.command.desired"
+                                 required
+                                 min="{{$ctrl.command.min}}"
+                                 max="{{$ctrl.command.max}}"/></div>
+  </div>
+
+</div>
+<div ng-if="!$ctrl.command.advancedMode">
+  <div class="form-group">
+    <div class="col-md-12">
+      <!--<p>Sets min, max, and desired instance counts to the same value.</p>-->
+
+      <!--<p> To allow true auto-scaling, use the <a href ng-click="$ctrl.command.advancedMode = true">Advanced Mode</a>.</p>-->
```

Comment:
Any reason to not allow setting different values for min/max/desired?

Reply:
@anotherchrisberry We started out with no autoscaling, and then when we added autoscaling we forgot about this part here.  I am reenabling it in an update to this PR